### PR TITLE
Added missing headers on user packages page

### PR DESCRIPTION
**Pull request by nagos**
User packages page had missing tabs.

packages-2
packages-1

### CODE REVIEW

**Comment by lafriks** [comment]
Isn't this template shared also with organizations that shouldn't have these tabs?

**Comment by nagos** [comment]
Template is shared, but this patch affects only user portion.

```
{{if .ContextUser.IsOrganization}}
...
{{else}}
...
```

https://github.com/go-gitea/gitea/pull/21172/files#diff-8020089c79cc846f287e738cf57917d21ec16499df1d994e6c31309d06a1a5afR54

packages-org

**Review by lunny — APPROVED** [review]

File: templates/user/overview/header.tmpl [hunk 1]
```diff
@@ -56,6 +56,22 @@
 						</a>
 					</div>
 				{{end}}
+			{{else}}
+				<a class='{{if eq .TabName "activity"}}active{{end}} item' href="{{.ContextUser.HomeLink}}?tab=activity">
+					{{svg "octicon-rss"}} {{.locale.Tr "user.activity"}}
+				</a>
+				{{if not .DisableStars}}
+					<a class='{{if eq .TabName "stars"}}active{{end}} item' href="{{.ContextUser.HomeLink}}?tab=stars">
```
Comment (lafriks) [inline templates/user/overview/header.tmpl]: Please add space between active and item inside if end

File: templates/user/overview/header.tmpl [hunk 2]
```diff
@@ -56,6 +56,22 @@
...
+						{{svg "octicon-star"}} {{.locale.Tr "user.starred"}}
+						{{if .Owner.NumStars}}
+							<div class="ui primary label">{{.Owner.NumStars}}</div>
+						{{end}}
+					</a>
+				{{else}}
+					<a class='{{if eq .TabName "watching"}}active{{end}} item' href="{{.ContextUser.HomeLink}}?tab=watching">
```
Comment (lafriks) [inline templates/user/overview/header.tmpl]: Same here

File: templates/user/overview/header.tmpl (hunk 1 above)
Comment (nagos) [reply templates/user/overview/header.tmpl]: Done

**Comment by nagos** [comment]
Space moved. But this code was copied from profile template, there are same problem.

https://github.com/go-gitea/gitea/blob/main/templates/user/profile.tmpl#L117

**Review by wolfogre — APPROVED** [review]

**Comment by lunny** [comment]
make L-G-T-M work

**Pull request merged**